app/views/declaration/previousDocuments/previous_documents_summary.scala.html

```diff
@@ -137,7 +138,7 @@
             html = HtmlFormat.fill(immutable.Seq(
```

Comment:
If you replace `Seq` with `List` then you do not need to import `scala.collection.immutable`

Reply:
Ok good to know but my ticket didn't touch this section of the code so probably out of scope of this review.